frontend/src/components/UserMenu.module.css

```
@@ -87,12 +78,42 @@
+  color: #fff;
+}
+
+/* スクリーンリーダー専用テキスト（視覚的には非表示） */
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)